scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -233,6 +233,101 @@ task SplitIntervals {
   }
 }
 
+task SplitIntervalsTarred {
+  input {
+    File intervals
+    File ref_fasta
+    File ref_fai
+    File ref_dict
+    Int scatter_count
+    File? interval_weights_bed
+    String? intervals_file_extension
+    String? split_intervals_extra_args
+    Int? split_intervals_disk_size_override
+    Int? split_intervals_mem_override
+    String? output_gcs_dir
+    String gatk_docker
+    File? gatk_override
+  }
+  meta {
+    # Not `volatile: true` since there shouldn't be a need to re-run this if there has already been a successful execution.
+  }
+
+  Int disk_size = select_first([split_intervals_disk_size_override, 10])
+  Int disk_memory = select_first([split_intervals_mem_override, 16])
```

Comment:
I think `disk_memory` is maybe supposed to be `memory_size`?

Reply:
Actually yes, it probably should be.  I just copied that from SplitIntervals.  Should I change it in both places?

Reply:
yes please! 🙏